src/AlertBar/Icon.js

```diff
@@ -0,0 +1,49 @@
+import React, { isValidElement } from 'react'
+import propTypes from 'prop-types'
+import { Valid, Warning, Error, Info } from '../icons/Status'
+import { spacers } from '../theme.js'
+
+const Icon = ({ icon, success, warning, critical }) => {
+    if (icon === false) {
+        return null
+    }
+
+    let IconComponent
+    if (isValidElement(icon)) {
```

Comment:
Normally this would be perfectly fine to use it like this, but as there's also a `<Valid />` component, I'd like to see `React.isValidElement(icon)` here instead (just to reduce the confusion)